build/ci/macos/build.sh

```diff
@@ -2,4 +2,11 @@
 
 echo "Build MuseScore"
 
-make -f Makefile.osx ci  
\ No newline at end of file
+BUILD_MODE=$1
```

Comment:
Do we really need the separate arg here?
Isn't it enough to have only BUILD_UI_MU4 option?

Reply:
This is arg for bash script and BUILD_UI_MU4 option for cmake.
I can't just pass the cmake options through the bash script.
Now the meaning is this, in the bash script is set by the argument in which mode to build. In the bash script, this is implemented by substituting the necessary cmake options.
In general, there may be a more when one option for some modes.